src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_ecc.c

```diff
@@ -84,6 +84,9 @@ uint64_t AppleCryptoNative_EccGetKeySizeInBits(SecKeyRef publicKey)
     // Word of caution: While seeking meaning in these numbers I ran across a snippet of code
     // which suggests that on iOS (vs macOS) they use a different set of reasoning and produce
     // different numbers (they used (8 + 2*thisValue) on iOS for "signature length").
+    //
+    // Starting with macOS Mojave and the new SecCertificateCopyKey API the values
+    // are the actual key size in bytes.
```

Comment:
Is there a way that we can tell which keys are which?   For the new ones the answer is just to multiply by 8.

Reply:
There possibly is a way, but I haven't figured it out yet. Naïvely I can check it based on presence of SecCertificateCopyKey, but I believe I still see the old values on the new system in some cases. I'll have to dig deeper to figure out when it returns the new one and when it returns the old ones. Multiplying by 8 doesn't always work because of padding bits (eg. 521 key size).

Reply:
The old values are still hit on Mojave for the following stack trace:

```
     at System.Security.Cryptography.EccSecurityTransforms.GetKeySize(SecKeyPair newKeys) in /Users/filipnavara/Documents/GitHub/corefx/src/Common/src/System/Security/Cryptography/EccSecurityTransforms.cs:line 175
     at System.Security.Cryptography.EccSecurityTransforms.ImportParameters(ECParameters parameters) in /Users/filipnavara/Documents/GitHub/corefx/src/Common/src/System/Security/Cryptography/EccSecurityTransforms.cs:line 166
     at System.Security.Cryptography.ECDsa.ECDsaImplementation.ECDsaSecurityTransforms.ImportParameters(ECParameters parameters) in /Users/filipnavara/Documents/GitHub/corefx/src/Common/src/System/Security/Cryptography/ECDsaSecurityTransforms.cs:line 207
     at System.Security.Cryptography.ECDsa.Create(ECParameters parameters) in /Users/filipnavara/Documents/GitHub/corefx/src/Common/src/System/Security/Cryptography/ECDsaSecurityTransforms.cs:line 44
```

In other words, it seems that keys created by `SecItemImport` (kSecFormatOpenSSL) and `SecCertificateCopyPublicKey` return the old values. Keys created by `SecCertificateCopyKey` return the new values.